libs/cli-kontrol-api/api/typescript/client/types.d.ts

```diff
@@ -154,6 +154,8 @@ export interface components {
       type: "gateway" | "service" | "service-version" | "redis";
```

Comment:
we may be missing some types here for postgres, etc right?

Reply:
I don't think we are using `redis` anymore.  This was used when things were hardcoded.  We could annotate more and then send that to the UI.